sdk/cognitivelanguage/Azure.AI.Language.Text.Authoring/samples/Sample16_TextAuthoring_AssignDeploymentResources.md

```diff
@@ -0,0 +1,39 @@
+# Assigning Deployment Resources Synchronously in Azure AI Language
+
+This sample demonstrates how to assign deployment resources synchronously using the `Azure.AI.Language.Text.Authoring` SDK.
+
+## Create an `AuthoringClient` using AAD Authentication
```

Comment:
Here we only have AAD, Let's have both Key and AAD in all pages. or just one common sample